assets/src/application/ui/display/icons/svgs/withEnhance.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import classNames from 'classnames';
+import { withSizeProps, IconSize } from './types';
+import './style.scss';
+
+const camelToSnakeCase = (str: string) => {
```

Comment:
@tn3rb let me know if this is smth that can be reused, such that it can be moved into a cross-cutting folder